refinery/tests/rusqlite.rs

```diff
@@ -210,32 +217,51 @@ mod rusqlite {
             .unwrap();
         assert_eq!(3, current);
 
-        let installed_on: DateTime<Local> = conn
+        let applied_on: DateTime<Local> = conn
             .query_row(
-                "SELECT installed_on FROM refinery_schema_history where version=(SELECT MAX(version) from refinery_schema_history)",
+                "SELECT applied_on FROM refinery_schema_history where version=(SELECT MAX(version) from refinery_schema_history)",
                 NO_PARAMS,
                 |row| {
-                    let _installed_on: String = row.get(0).unwrap();
-                    Ok(DateTime::parse_from_rfc3339(&_installed_on).unwrap().with_timezone(&Local))
+                    let applied_on: String = row.get(0).unwrap();
+                    Ok(DateTime::parse_from_rfc3339(&applied_on).unwrap().with_timezone(&Local))
                 }
             )
             .unwrap();
-        assert_eq!(Local::today(), installed_on.date());
+        assert_eq!(Local::today(), applied_on.date());
     }
 
     #[test]
     fn applies_new_migration() {
         let mut conn = Connection::open_in_memory().unwrap();
 
-        mod_migrations::migrations::runner().run(&mut conn).unwrap();
+        embedded::migrations::runner().run(&mut conn).unwrap();
 
-        let migration = Migration::from_filename(
+        let migration1 = Migration::from_filename(
+            "V1__initial.sql",
```

Comment:
I'm wondering if the library should maybe allow people to opt-into an opinionated mode where things like migration names (especially when it comes to creating new migrations) is more fixed and also uses dates instead of just numbers

Reply:
yeah, the rational for choosing numbers over dates, was the logic behind selecting a default Timezone, and how to deal with people working on a project in different Timezones versus the problem of having to rename and reword a migration on a merge conflict where different developers added the same migration version.
But maybe makes sense to add a feature that let's user's chose which method suits them better

Reply:
I think we could just UTC as a timezone. But yea, this doesn't mitigate the merge problems. Or rather, I think it's different merge problems.